servers/src/main/java/tachyon/Format.java

```diff
@@ -67,6 +67,18 @@ public static void main(String[] args) throws IOException {
         System.exit(-1);
       }
 
+      String[] masterServiceNames = new String[] {
+          Constants.BLOCK_MASTER_SERVICE_NAME,
+          Constants.FILE_SYSTEM_MASTER_SERVICE_NAME,
+          Constants.RAW_TABLE_MASTER_SERVICE_NAME,
+      };
```

Comment:
Share this constant String array `masterServiceNames` with `minicluster/src/main/java/tachyon/master/LocalTachyonMaster.java` listed above.